### ISSUE TITLE
Docker not working with Linux and Windows containers.

### ISSUE THREAD

**Issue by Ariwari** [docker/docker#31823]
**Description**

Following problems were encountered on a: 
Dell Lattitude E5450
Windows 10 Enterprise 64bit with Hyper-V and Containers features enabled
I5-5300U (supports vt-x)
8 GB RAM

**Steps to reproduce the issue:**
1. Install Docker
2. Switch to Windows containers (because the Linux containers do not work)
3. Encounter Windows container specific errors with "docker run -it microsoft/nanoserver cmd" in an elevated powershell:

** The problem **
Installed Docker for Windows last thursday.

First things first: starting up Docker with "Linux containers" gave me the following daemon error (crash log ID 59BD64BD-5B3F-4AFC-AD3D-89AB1E38ACF9/2017-03-14_15-13-10)

Unable to start: The running command stopped because the preference variable "ErrorActionPreference" or common parameter is set to Stop: Kan MobyLinuxVM niet starten. (Virtuelemachine-id DA92158C-A6A3-4973-A40F-F44D84B6EDEC)

MobyLinuxVM kan werkproces niet starten: Er is een ongeldige ingang opgegeven. (0xC0000008). (Virtuelemachine-id DA92158C-A6A3-4973-A40F-F44D84B6EDEC)
at Start-MobyLinuxVM, <No file>: line 287
at <ScriptBlock>, <No file>: line 382
   bij Docker.Backend.ContainerEngine.Linux.DoStart(Settings settings)
   bij Docker.Backend.ContainerEngine.Linux.Start(Settings settings)
   bij Docker.Core.Pipe.NamedPipeServer.<>c__DisplayClass8_0.<Register>b__0(Object[] parameters)
   bij Docker.Core.Pipe.NamedPipeServer.RunAction(String action, Object[] parameters)

The log in the crash log will tell you that Hyper-V seems to create the MobyLinuxVM and for some reason remove it a couple of seconds later just before throwing the error. This is also visible in the Hyper-V manager: a new MobyLinuxVM pops up and disappears a few seconds later.

------------------------------
Now onto trying Windows Containers:

At least Docker manages to startup now, so far so good.

Until I try to run "docker run -it microsoft/nanoserver cmd" in an elevated powershell:

C:\Program Files\Docker\Docker\Resources\bin\docker.exe: Error response from daemon: container 6436ba30cee8c90e2a9f415b4
db5ad5083c3d921fc4d7e75f8591c33a17fb844 encountered an error during CreateContainer: failure in a Windows system call: w
inapi error #3221225480 (0xc0000008) extra info: {"SystemType":"Container","Name":"6436ba30cee8c90e2a9f415b4db5ad5083c3d
921fc4d7e75f8591c33a17fb844","Owner":"docker","IsDummy":false,"IgnoreFlushesDuringBoot":true,"LayerFolderPath":"C:\\Prog
ramData\\Docker\\windowsfilter\\6436ba30cee8c90e2a9f415b4db5ad5083c3d921fc4d7e75f8591c33a17fb844","Layers":[{"ID":"ce547
0b9-4d9a-572b-b137-fc113e95928d","Path":"C:\\ProgramData\\Docker\\windowsfilter\\2beafeda2d5096e4751c3b22d82979ecda9b4a0
83525a540e43366ed059bd35b"},{"ID":"b8306d45-c24b-54a2-b061-dd0ce8970f06","Path":"C:\\ProgramData\\Docker\\windowsfilter\
\e81ee89bb3d705b44f750e0b01d7ca3c0c4fcdc1228236969774671da3ea5c3a"}],"HostName":"6436ba30cee8","MappedDirectories":[],"S
andboxPath":"C:\\ProgramData\\Docker\\windowsfilter","HvPartition":true,"EndpointList":["e984bd54-793e-48ce-b7e3-0c6438b
bb35f"],"HvRuntime":{"ImagePath":"C:\\ProgramData\\Docker\\windowsfilter\\2beafeda2d5096e4751c3b22d82979ecda9b4a083525a5
40e43366ed059bd35b\\UtilityVM"},"Servicing":false,"AllowUnqualifiedDNSQuery":true}.

That won't work either.

Any help?

[log.txt](https://github.com/docker/docker/files/841800/log.txt)

**Comment by friism** [comment]
@Ariwari for the Linux problem, can you please open a separate issue on the Docker for Windows issue tracker (please include the diagnostic id)? https://github.com/docker/for-win/issues/new

I haven't seen `Windows system call: winapi error #3221225480` - maybe @jhowardmsft has ideas.

**Comment by jhowardmsft** [comment]
I'd need the analytic logs out of Hyper-V compute as the next step to see what might be going on there on the Windows daemon side.

**Comment by friism** [comment]
@jhowardmsft can you summarize how to do that for @Ariwari ?

**Comment by jhowardmsft** [comment]
Win+R 
eventvwr
Expand to Applications and Services\Microsoft\Windows\Hyper-V-Compute\*
View Analytic logs from menu
Wait for it to expand...
Select analytic
Right-click enable
Ignore warning about possible missed events
<repro>
Disable log
Save .evtx file (along with same for admin/operational) somewhere

**Comment by Ariwari** [comment]
Hello, thansk for the speedy response!

I've attached the logging to this comment as a zip.

**Comment by Ariwari** [comment]
Hello, thansk for the speedy response!

I've attached the logging to this comment as a zip.

(p.s I closed the thread on accident, this was not my intention)

**Comment by Ariwari** [comment]
@friism Made a new issue as you asked.

**Comment by Ariwari** [comment]
Fixed the problem.

Turned out to be a (yet unknown) conflict between the Windows Anniversary Update and my the policy my company has.
